packages/webapp/src/components/forms/AddClientForm/index.tsx

```diff
@@ -105,6 +107,7 @@ export const AddClientForm: StandardFC<AddClientFormProps> = wrap(function AddCl
 			closeForm()
 		} else {
 			setSubmitMessage(response.message)
+			setSubmitButtonDisabledState(false)
```

Comment:
Does the form get emptied once it has been submitted?

Reply:
Yes, once `closeForm()` is called, the component is removed and when it is reopened `isSubmitButtonDisabled` is reset to `false`